client/src/util/dataframe/dataframe.js

```diff
@@ -135,97 +141,107 @@ class Dataframe {
     }
   }
 
-  __compile() {
+  static __compileColumn(column, getOffset, getLabel) {
     /*
```

Comment:
[Nit] Inconsistent spacing with block level comments. Some are inline with the `/*`, some are indented, sometimes the closing `*/` is indented.

Not a big deal but it can cause extra changes in the diff (see below) if the comment changes.

Reply:
done

Reply:
__compileColumn is still indented while all others are not